arnab_bhadra/arnabbhadra_PostgresAssignment/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "arnabbhadra_postgresassignment",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "node ./src/server.js"
```

Comment:
You can also add other scripts. Follow https://github.com/adityajawade93/freshers-june2021/tree/main/jatin/postgres_assignment